src/dash-table/components/CellFactory.tsx

```diff
@@ -51,7 +50,8 @@ export default class CellFactory {
             style_cell_conditional,
             style_data,
             style_data_conditional,
-            virtualized
+            virtualized,
+            visibleColumns
```

Comment:
`columns` became `visibleColumns` pretty much everywhere under the sanitation layer. Updating usage / destructuring-ordering accordingly.

Reply:
columns -> visibleColumns
IVisibleColumn -> IColumn
etc. on that theme